T02/Program.cs

```diff
@@ -0,0 +1,138 @@
+         var (remWord, remRoman) = (words[rem], romans[rem]);
+         bool remZero = rem is 0;
+         var val = int.Parse (rem is 1 ? $"{rem}{prev}" : $"{rem}0");
+         (s, todo) = (s, remZero) switch {
+            (A, true) => (B, none),
+            (A, false) => (B, () => { word.Insert (0, remWord); roman.Append (remRoman); prev = rem; last = rem; }),
+            (B, true) => (C, none),
+            (B, false) => (C, () => {
+               if (rem is 1) word.Clear (); word.Insert (0, words[val] + " ");
+               switch (val) {
+                  case 40 or 90: roman.Insert (0, romans[val]); break;
+                  case >= 50 and < 90:
+                     for (int j = 0, k = (val - 50) / 10; j < k; j++) roman.Insert (0, "X"); roman.Insert (0, "L"); break;
+                  default: for (int j = 0, k = val / 10; j < k; j++) roman.Insert (0, "X"); break;
+               }
+               last = rem;
+            }
+            ),
+            (C, true) => (D, none),
+            (C, false) => (D, () => {
+               var extnd = last is 0 ? "" : "and"; word.Insert (0, $"{remWord} Hundred {extnd} ");
+               switch (val) {
+                  case 40 or 90: roman.Insert (0, romans[val * 10]); break;
+                  case < 50: for (int j = 0, k = val / 10; j < k; j++) roman.Insert (0, "C"); break;
+                  default: for (int j = 0, k = (val - 50) / 10; j < k; j++) roman.Insert (0, "C"); roman.Insert (0, "D"); break;
+               }
+            }
+            ),
+            (D, true) => (E, () => prev = rem),
+            (D, false) => (E, () => {
+               word.Insert (0, $"{remWord} Thousand "); prev = rem;
+               for (int j = 0, k = val / 10; j < k; j++) roman.Insert (0, "M");
+            }
+            ),
+            _ => (G, none)
+         };
+         todo ();
+      }
+      return (word.ToString (), roman.ToString ());
+   }
+
+   static void ToWordAndRoman (int n) {
+      var (word, roman) = Convert (n);
+      WriteLine ($"Number\t: {n}\n");
+      ForegroundColor = ConsoleColor.Green;
+      WriteLine ("Conversion By Digits Count");
+      ResetColor ();
+      WriteLine ($"Word\t: {GetWord (n)}\nRoman\t: {GetRoman (n)}\n");
+      ForegroundColor = ConsoleColor.Green;
+      WriteLine ("Conversion By State Machines");
+      ResetColor ();
+      WriteLine ($"Word\t: {word}\nRoman\t: {roman}");
+      WriteLine ("_________________________________________");
+   }
+
+   /// <summary> Returns word format of given number </summary>
+   static string GetWord (int n) {
+      var (word, str) = ("", n.ToString ());
+      var (len, firstDigit, lastDigit) = (str.Length, int.Parse (str[0].ToString ()), n % 10);
+      Action todo = () => { };
+      todo = len switch {
+         2 => () => {
+            var tmp = firstDigit is 0 || lastDigit is 0 || n is >= 10 and <= 20 ? words[n] : $"{words[firstDigit * 10]} {words[lastDigit]}";
+            word = tmp;
+         }

```

Comment:
```suggestion
         },

```
Same here braces and comma can be on the same line